src/components/FormDropdown/FormDropdown.vue

```diff
@@ -80,8 +122,48 @@ const label = computed(() => {
     return props.modelValue[props.labelKey];
 });
 
-const {classes} = useClasses(computed(() => [
-    ...useFormSelect(props).classes.value,
-    props.toggleClass,
-]));
+const propsDropdown = computed(() => {
+    const properties = {};
+
+    Object.keys(dropdownProps)
+        .forEach(k => properties[k] = props[k]);
+
+    return properties;
+});
+
+const searchQuery = ref(props.searchQuery);
```

Comment:
Not possible. This will only make a reference to the prop. This will mean if we would `searchQuery.value = 'test'` we are mutating a prop. I already update the value of `searchQuery` when `props.searchQuery` changes in a watcher.